app/members/page.tsx

```diff
@@ -0,0 +1,32 @@
+import GitHubCorner from "@/components/GitHubCorner/GitHubCorner";
+import styles from "./members.module.css";
+import Members from "@/components/Members/Members";
+import getAllUsers from "@/lib/getAllUsers";
+import Link from "next/link";
+
+export default async function MembersPage() {
+  const usersData: Promise<User[]> = getAllUsers();
+  const users = await usersData;
+  console.log("Hello");
```

Comment:
I would say we can delete this console.log 😄

Reply:
@amandhawb Hi Amanda, yes! I was experimenting with some code for our members page here and it will be deleted! We are trying to figure out how to render local JSON data onto our Members page and this was an experimental approach.